### ISSUE TITLE
"Expand all" / "Collapse all" buttons ?

### ISSUE THREAD

**Issue by castellfr** [coveooss/json-schema-for-humans#11]
Hi, very useful tool, thanks.
Is there any plan to add some buttons to expand/collapse the whole json schema structure in the output html?

**Comment by dblanchette** [comment]
Hi!

Would that be the whole structure recursively or just the top level elements? I was able to add the whole structure by adding those buttons:

```html
    <button class="btn btn-primary" type="button" data-toggle="collapse" data-target=".collapse:not(.show)" aria-expanded="false">Expand all</button>
    <button class="btn btn-primary" type="button" data-toggle="collapse" data-target=".collapse.show" aria-expanded="false">Collapse all</button>
```

It works well with relatively small schemas, but with big ones it freezes the browser.

I could always add that as an option though

**Comment by castellfr** [comment]
I think recursively for the whole thing as an option is great. Thanks !

**Comment by dblanchette** [comment]
Released in version 0.5

**Comment by castellfr** [comment]
Thanks !
